src/Cake.Core/ICakeEnvironment.cs

```diff
@@ -51,5 +52,11 @@ public interface ICakeEnvironment
         /// </summary>
         /// <returns>The environment variables as IDictionary&lt;string, string&gt; </returns>
```

Comment:
Should be something like:

`<returns>The environment variables like <see cref="IDictionary{string, string}" />.`

(Not 100% sure about the syntax for generic arguments but I think that's correct)